com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -868,6 +868,13 @@ void CreateCameraRenderTarget(ScriptableRenderContext context, ref RenderTexture
                     colorDescriptor.autoGenerateMips = false;
                     colorDescriptor.depthBufferBits = (useDepthRenderBuffer) ? k_DepthStencilBufferBits : 0;
                     cmd.GetTemporaryRT(m_ActiveCameraColorAttachment.id, colorDescriptor, FilterMode.Bilinear);
+#if ENABLE_VR && ENABLE_XR_MODULE && UNITY_WSA_10_0 && UNITY_64
```

Comment:
Need these preprocs to make sure this does not happen on tiled renderers. It ruins performance on quest and hololens. Also this bug does not occur on those platforms.

Reply:
any idea why only this platform has the bug? could it be a deeper issue on the C++ side for this API ?

Reply:
In the PIX capture we see C++ calls DiscardResource. The call after is ClearRenderTargetView clears to colour yellow which C++/C# does not call. It seems the driver adds that call.

Also Unity docs state:
You can not depend on any particular contents of the RenderTexture you get from GetTemporary function. It might be garbage, or it might be cleared to some color, depending on the platform.

Reply:
Is it possible to fit in this XR specific clear logic into the URP clear flag function here: https://github.com/Unity-Technologies/Graphics/blob/6bcd723e6833becd1d68457a5b7bc991f23dba7e/com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs#L763 ?
I think it would be great we can put all clear related logic in one place for easier maintiance.